web/src/features/fireWeather/components/tables/SortableTableByDatetime.tsx

```diff
@@ -65,6 +105,24 @@ interface Props<R> {
   columns: Column[]
 }
 
+interface MinMaxValues {
+  relative_humidity: number | null
+  precipitation: number | null
+  wind_speed: number | null
+  temperature: {
+    min: number | null
+    max: number | null
+  }
+}
+
+interface RowIdsOfMinMaxValues {
+  relative_humidity: number[]
+  precipitation: number[]
+  wind: number[]
+  max_temp: number[]
+  min_temp: number[]
+}
+
 function SortableTableByDatetime<R extends WeatherValue>(props: Props<R>) {
```

Comment:
A unit test suite for this component may be in order due to all the conditional branches and switches -- I'm not sure the cypress tests capture that and I think unit tests would be easier to write. WDYT?